### ISSUE TITLE
Is it possible to replace assetic by webpack-encore ?

### ISSUE THREAD

**Issue by craph** [avanzu/AdminThemeBundle#229]
In using Symfony 4 for my project and I would like to use AdminLTE as admin theme.
In symfony documentation, they indicate that starting from Symfony 2.8, Assetic is no longer included by default in the Symfony Standard Edition [Symfony 3]( https://symfony.com/doc/3.4/frontend/assetic.html) and [Symfony 4](https://symfony.com/doc/current/frontend/assetic.html). 

FEATURE REQUEST:

  * I want to know if you have any process that enables us to replace assetics by wepack-encore while configuring adminLTE.

### Debug info

| Symfony version  |  4.0  |
| AdminThemeBundle  | latest  |

**Comment by kevinpapst** [comment]
Yes there is, but it doesn't ship out of the box and it was a bit of a pain to set it up...

You can check out my solution (which is fas from being perfect, but it works).
This project is using this bundle with Symfony 4 and webpack encore:
https://github.com/kevinpapst/kimai2

**Comment by craph** [comment]
Thank you very much :) i will check it ^^
